Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Docker 17.05 or higher required for multi-stage builds
+FROM rust:1.26.0-stretch as builder
+ADD . /app
+WORKDIR /app
+# Make sure that this matches in .travis.yml
+ARG RUST_TOOLCHAIN=nightly
+RUN \
+    apt-get -qq update && \
+    apt-get -qq install -y default-libmysqlclient-dev && \
+    \
+    rustup default ${RUST_TOOLCHAIN} && \
+    cargo --version && \
+    rustc --version && \
+    mkdir -m 755 bin && \
+    cargo build --release && \
+    cp /app/target/release/service /app/bin
+
+
+FROM debian:stretch-slim
+# FROM debian:stretch  # for debugging docker build
+RUN \
+    groupadd --gid 10001 app && \
+    useradd --uid 10001 --gid 10001 --home /app --create-home app && \
+    \
+    apt-get -qq update && \
+    apt-get -qq install -y default-libmysqlclient-dev libssl-dev && \
+    rm -rf /var/lib/apt/lists
+
+COPY --from=builder /app/bin /app/bin
+
+WORKDIR /app
+USER app
+
+# override rocket's dev env defaulting to localhost
+ENV ROCKET_ADDRESS 0.0.0.0
+
+CMD ["/app/bin/service"]
```

Comment:
This is interesting, we need 2 processes here

Reply:
Except we probably don't want the `queues` process running automatically on every box, it should be configurable. Not sure how straightforward that is, but if it's a problem maybe we just have a separate `Dockerfile` for a box that will be spun up to process the queues.

Reply:
>  we probably don't want the queues process running automatically on every box

We had trouble figuring this out in the past where only certain boxes should have certain processes. I'm gonna add this to dev ops meeting notes to chat about